File: C/String/program22.c
//Accept two string from user and concat first string after the second string

#include<stdio.h>

void strcatX(char *src, char *dest)
{
    while(*src != '\0')
    {
        src++;
    }

    while(*dest != '\0')
    {
        *src = *dest;
        src++;
        dest++;
    }
    *src = '\0';
}

int main()
{
    char Arr[20];
    char Brr[20];

    printf("Enter first string: \n");
    scanf("%[^'\n']s",Arr);

    printf("Enter second string: \n");
    scanf(" %[^'\n']s",Brr);

    strcatX(Arr,Brr);
    printf("String after concatination : %s ",Arr);

    return 0;
}

File: Java/String/program22.java
//Accept the string from user and reverse that string

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public void strrevX()
    {
        char Arr[] = str.toCharArray();

        int i = 0;

        System.out.println("String after revserse : ");

        for(i = Arr.length-1; i >= 0; i--)
        {
            System.out.print(Arr[i]);
        }      
    }
}

class program22
{
    public static void main(String arg[])
    {
        Stringx obj = new Stringx();

        obj.Accept();

        obj.strrevX();
    }
}